### ISSUE TITLE
Web UI should *not* use modal dialogs

### ISSUE THREAD

**Issue by riccardomurri** [TissueMAPS/TissueMAPS#79]
Modal dialogs (i.e., those who require that the user clicks on a button to dismiss the notification) are generally an annoyance. In TM's UI they are actually worse: since workflow status pages are implementing by reloading/polling the server for state, whenever there is a network- or server-side problem, dialog boxes will stack up a dozen a minute, until there are so many that it is impossible to get rid of all of them and one must kill the browser tab and open a new one.

Again, there's a few related issues that should be fixed:

1. Modal dialogs should *not* be used for notifications!  Just have a small drop-down box that disappears after a few seconds.
2. Notifications with the exact same content should not be shown (unless some relatively large amount of time has passed)
3. Modal dialog boxes should *only* be used for actions that really require a confirmation (e.g., "Kill workflow")

**Comment by hackermd** [comment]
I agree! Model dialog boxes are also a pain in Angular with respect to caching, as we have observed with figures generated by Jterator modules.
